scss/style.scss

```diff
@@ -2,5 +2,145 @@
 
 @import "variables";
 @import "typography";
-
 // import components here
+@font-face {
+    font-family: Raleway; 
+    src: url(src/fonts/Raleway-SemiBold.ttf); 
+   }
+* {
+  margin: 0;
+  padding: 0;
+  font-family: Raleway, sans-serif;
+}
+
+a {
+    text-decoration: none;
+    color:#FAFAFA;
+}
+body {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+}
+.start-screen-block {
+  width: $container-width;
+  height: 900px;
+  border: 1px solid black;
+  background: #3d4148;
+}
+header {
+  position: absolute;
+  margin: 20px 230px 0px;
+  width: 1021px;
+  height: 465px;
+  @extend %centering;
+  border: 1px solid blue;
+}
+header ul{
+    display: flex;
+    justify-content: space-between;
+    list-style-type:none;
+    width: 963px;
+    height: 25px;
+    margin-top:38px;
+    margin-left:-35px;
+}
+header li{
+    float:left;
+    display: block;
+    
+    font-size: 18px;
+    color:#FAFAFA;
+}
+header li:hover{
+    border-bottom:1px solid red;
```

Comment:
It's better to make
```
header li {
  border-bottom:1px solid transparent;
  ...
}

header li:hover {
  border-bottom-color: red;
}
```

Reply:
And you shouldn't write colors by names, use hex code or rgb/rgba

Reply:
Done. got you